src/backend/src/utils/validation.utils.ts

```diff
@@ -197,6 +197,21 @@ export const projectValidators = [
   nonEmptyString(body('managerId').optional())
 ];
 
+export const materialValidators = [
+  nonEmptyString(body('name')),
+  nonEmptyString(body('assemblyId').optional()),
+  isMaterialStatus(body('status')),
+  nonEmptyString(body('materialTypeName')),
+  nonEmptyString(body('manufacturerName')),
+  nonEmptyString(body('manufacturerPartNumber')),
+  nonEmptyString(body('pdmFileName').optional()),
+  decimalMinZero(body('quantity')),
+  nonEmptyString(body('unitName')).optional(),
+  intMinZero(body('price')), // in cents
+  intMinZero(body('subtotal')), // in cents
+  nonEmptyString(body('linkUrl')),
```

Comment:
You got rid of the isURL() validation -- is that intended?

Reply:
yeah it wasnt working. also doesnt really matter

Reply:
interesting - seemed to work for me... but  getting rid of it ok ill resolve